models/aeif_cond_alpha_RK5.cpp

```diff
@@ -179,35 +185,49 @@ nest::aeif_cond_alpha_RK5::Parameters_::set( const DictionaryDatum& d )
   if ( updateValue< double >( d, names::MAXERR, tmp ) )
   {
     if ( not( tmp > 0.0 ) )
+    {
       throw BadProperty( "MAXERR must be positive." );
+    }
     MAXERR = tmp;
   }
 
   if ( updateValue< double >( d, names::HMIN, tmp ) )
   {
     if ( not( tmp > 0.0 ) )
+    {
       throw BadProperty( "HMIN must be positive." );
+    }
     HMIN = tmp;
   }
 
   if ( Delta_T != 0. && V_peak_ <= V_th )
+  {
     throw BadProperty( "V_peak must be larger than threshold." );
+  }
   else if ( Delta_T == 0. )
+  {
     updateValue< double >( d, names::V_peak, V_th ); // expected behaviour
+  }
 
   if ( V_reset_ >= V_peak_ )
+  {
     throw BadProperty( "Ensure that: V_reset < V_peak ." );
+  }
 
   if ( C_m <= 0 )
   {
     throw BadProperty( "Capacitance must be strictly positive." );
   }
 
   if ( t_ref_ < 0 )
+  {
     throw BadProperty( "Refractory time cannot be negative." );
+  }
 
   if ( tau_syn_ex <= 0 || tau_syn_in <= 0 || tau_w <= 0 )
+  {
     throw BadProperty( "All time constants must be strictly positive." );
+  }
```

Comment:
@Silmathoron Now I managed to confuse myself. Was this about adding braces? Anyways, the code looks fine here.